util/deploy-production.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Helper script for deploying to production.
+# Needs the following packages to be installed: google-cloud-cli, gh, jq
+
+set -e
+
+usage() {
+  USAGE="Usage: deploy-production.sh [-f]
+    -f : Always deploy (even if checks have failed)"
+  echo "${USAGE}"
+}
+
+while getopts ':fh' flag; do
+  case "${flag}" in
+    f) FORCE_DEPLOY='true' ;;
+    h|*) usage && exit 0;;
+  esac
+done
+
+GH_OWNER="web-platform-tests"
+GH_REPO="wpt.fyi"
+
+# Find changes to deploy.
+LAST_DEPLOYED_SHA=$(gcloud app --project=wptdashboard versions list --hide-no-traffic --filter='service=default' --format=yaml | grep id | head -1 | cut -d' ' -f2 | sed 's/rev-//')
+CHANGELIST=$(git log $LAST_DEPLOYED_SHA..HEAD --oneline)
+if [[ "${CHANGELIST}" == ""  ]];
+then
+    echo "No new changes to deploy."
+    exit 0
+fi
+CHANGE_COUNT=$(echo "$CHANGELIST"|wc -l)
+echo -e "There are $CHANGE_COUNT changes to deploy:\n$CHANGELIST"
+
+# Verfiy that all commit checks passed.
+FAILED_CHECKS=$(gh api /repos/"$GH_OWNER"/"$GH_REPO"/commits/HEAD/check-runs | jq -r '.check_runs | map(select(.conclusion == "failure"))')
+FAILURES=$(echo "$FAILED_CHECKS" | jq -r 'length')
+if [[ "${FAILURES}" != ""  ]];
```

Comment:
Given FAILURES is an int, shouldn't this be the following:

```
if [[ "${FAILURES}" != "0"  ]];
```

Reply:
```sh
$ FAILURES=0
$ if [[ "${FAILURES}" != ""  ]]; then  echo "Found $FAILURES failures. EXITING!"; else echo "no failures found"; fi
Found 0 failures. EXITING!


# Modify the conditional
$ if [[ "${FAILURES}" != "0"  ]]; then  echo "Found $FAILURES failures"; else echo "no failures found"; fi
no failures found

```

Reply:
Indeed, good catch. Fixed.